Microsoft.Toolkit.Uwp.UI.Media/Pipelines/PipelineBuilder.Effects.cs

```diff
@@ -476,8 +476,8 @@ public PipelineBuilder Tint(Color color, out EffectAnimation<Color> animation)
         /// <summary>
```

Comment:
I think you forgot to add the ranges for the Exposure methods.

Reply:
Ah, right, thanks! 😊

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3304/commits/86b80d1697b43a5a51edb7e789215e1edfaeb053.